src/components/PostsTrendingCard/PostsTrendingCard.scss

```diff
@@ -0,0 +1,29 @@
+@import '~styles/main';
+
+
+.postsTrendingCard{
+  @include flex-item-center(row);
+  width: 594px;
+
+  &__img {
+    height: 141px;
+}
+
+  &__content {
```

Comment:
There is already a mixin that creates an element (BEM), you can use that to create the element.
The code for creating element must look like: `@include e('content') { ... }`

Reply:
Done